pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -126,8 +130,8 @@ while [ "$#" -gt 0 ]; do
     esac
 done
 
-if [ -n "$SUDO_USER" ]; then
-    maybeSudo=(sudo --)
+if [[ -n "$SUDO_USER" || -n $remoteSudo ]]; then
```

Comment:
```suggestion
if [ -n "$SUDO_USER" ] || [ -n $remoteSudo ]; then
```
Just a suggestion to avoid it relying on bashisms, remaning sh/posix compatible =)

Reply:
There is no advantage in being posix compatible except that we easier create bugs. Nix always runs in bash and heavily relies on that. Making everything posix compatible will be hard and it is most likely easier to boostrap bash on a new platform.

Reply:
That's fair enough, thank you for enlightening me! I thought default of `sh` was [ash](https://linux.die.net/man/1/ash), not `bash` you see =)

Feel free to ignore my suggestion!

Reply:
I had a look at that angle in the rest of the file. It's already a mix and has been for a few ~2 years so I landed on the sort of reasoning pointed out above.